pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -188,7 +188,8 @@ class Build {
 												context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
 												context.string(name: 'RELEASE_TAG', value: "${buildConfig.SCM_REF}"),
 												context.string(name: 'JDK_REPO', value: jdkRepo),
-												context.string(name: 'JDK_BRANCH', value: jdkBranch)]
+												context.string(name: 'JDK_BRANCH', value: jdkBranch)
```

Comment:
Missing comma at EOL

Reply:
Whoops. Thanks for catching that. Fixed.